README.md

```diff
@@ -13,13 +13,17 @@ We do not condone using this project as a means for piracy in any form. This pro
 * Added a built-in shader compiler for backends/platforms that support it.
 * Added various other backends to windows aside from the usual DirectX 9 backends
 
-## If you are here for Sonic Mania:
-You probably want [the dedicated Sonic Mania repo](https://github.com/Rubberduckycooly/Sonic-Mania-Decompilation) instead.
+## If you are here for Sonic Mania's source code:
+You probably want to head to [the Sonic Mania Decompilation repo](https://github.com/Rubberduckycooly/Sonic-Mania-Decompilation) instead.
```

Comment:
We can probably change this to allude to the fact you can build both from the Mania repo?